core/jvm/src/test/scala/fs2/CompressionSuite.scala

```diff
@@ -118,7 +118,7 @@ class CompressionSuite extends Fs2Suite {
           .chunk[IO, Byte](Chunk.bytes(getBytes(s)))
           .rechunkRandomlyWithSeed(0.1, 2)(System.nanoTime())
           .through(
-            deflate(
+            deflateParams(
```

Comment:
Dotty appears confused by some interaction of overloads and default arguments. Without this change, Dotty complains:

```scala
[error] -- [E134] Type Mismatch Error: /Users/mpilquist/Development/oss/fs2/core/jvm/src/test/scala/fs2/CompressionSuite.scala:231:19
[error] 231 |    val deflater = deflate[IO](bufferSize = chunkSize)
[error]     |                   ^^^^^^^
[error]     |None of the overloaded alternatives of method deflate in object compression with types
[error]     | [F[_$2]]
[error]     |  (deflate: fs2.compression.DeflateParams)
[error]     |    (implicit SyncF: cats.effect.Sync[F]): fs2.Pipe[F, Byte, Byte]
[error]     | [F[_$1]]
[error]     |  (level: Int, nowrap: Boolean, bufferSize: Int, strategy: Int)
[error]     |    (implicit SyncF: cats.effect.Sync[F]): fs2.Pipe[F, Byte, Byte]
[error]     |match type arguments [cats.effect.IO] and arguments ((chunkSize : Int))
```